great_expectations/validator/validator.py

```diff
@@ -2162,6 +2162,33 @@ def head(
 
         return df.reset_index(drop=True, inplace=False)
 
+    @staticmethod
+    def is_expectation_self_initializing(name: str) -> bool:
+        """
+        Given the name of an Expectation, returns a boolean that represents whether an Expectation can be auto-intialized.
+
+        Args:
+            name (str): name of Expectation
+
+        Returns:
+            boolean that represents whether an Expectation can be auto-initialized. Information also outputted to logger.
```

Comment:
We're printing instead of logging right? Do we always want to emit this information to the user? If not, let's just a log at an appropriate level.

Also, does this have to go on the validator? Can it maybe be a static method on the Expectation class? Just brainstorming but I know the validator is quite bloated.

Reply:
Great points. 

* I'd like to propose printing instead of logging. I think the most common usecase for this would be in a Notebook, and typically `warning` and above is displayed in Jupyter Notebooks by default.. which I dont think would be appropriate in this case. 

* Regarding your point about Expectation, I tried this yesterday and ran into some circular imports .. I can take another stab at it before merging (I think it makes more sense to have it at the `Expectation` level too)